.travis-ci.sh

```diff
@@ -1,4 +1,5 @@
 eval `opam config env`
+sudo apt update
```

Comment:
Oh wait, it has to be added to a `make depend` target that's generated by mirage itself.